src/main/java/seedu/address/logic/commands/FavoriteCommand.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands;
+
+import java.util.Collections;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.ReadOnlyPerson;
+import seedu.address.model.person.exceptions.DuplicatePersonException;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+
+/**
+ * Favorites the person(s) identified using it's last displayed index from the address book.
+ */
+public class FavoriteCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "fav";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Favorites the person(s) identified by the index number used in the last person listing.\n"
+            + "Parameters: INDEX (must be a positive integer) or INDEXES\n"
```

Comment:
We can represent this as INDEX [ADDITIONAL INDEXES]

Reply:
Updated ✔